website/docs/r/spot_fleet_request.html.markdown

```diff
@@ -78,6 +78,91 @@ resource "aws_spot_fleet_request" "foo" {
 }
 ```
 
+### Example with a dynamic block
```

Comment:
For these example sub-headings we try to identify use-cases (the intended goal) rather than specific language features (means if reaching that goal).

I don't know this AWS feature particularly well, but it looks like the use-case here is something like "launching multiple instance types across many subnets". Does that seem right?

Reply:
I spent quite a few hours trying to find a working example, and the closest thing was your bit of code I mentioned on slack.
the problem with this awes resource is that `launch_specification` will be repeated per each property.
on tf v0.11 I have a offline generated 6000 lines of code block to expand all subnets per instance size.
with 0.12 and a `for_each`I can easily do that in a single block... especially with the `setproduct ` to create the matrix for me
pretty much any user of this resource will need to expand at least per subnet, if not per instance size like I did.
Having a good sample example can save people hours trying to find a working way to do a `for_each`. 
that is the point of my PR.

Reply:
I would appreciate some guidance here

Reply:
In line with my other comment of simplifying this to just be over subnets, perhaps the following heading:

```markdown
# Launch Specifications in Multiple Subnets
```

Reply:
Perfect.
I'll try to clean up my extra stuff tomorrow

Reply:
I've simplify this a lot
not sure i need `s[1]` there anymore